src/main/scala/com/fulcrumgenomics/umi/CallDuplexConsensusReads.scala

```diff
@@ -108,6 +109,12 @@ class CallDuplexConsensusReads
  val maxReadsPerStrand: Option[Int] = None,
  @arg(doc="The number of threads to use while consensus calling.") val threads: Int = 1,
  @arg(doc="Consensus call overlapping bases in mapped paired end reads") val consensusCallOverlappingBases: Boolean = true,
+ @arg(doc="""
+            |Pull reads from this many source molecules into memory for multi-threaded processing.
```

Comment:
not strictly true, since the `ParIterator` will have `threads * maxSourceMoleculesInMemory`, and so will the `AsyncIterator` that wraps it, so it should be `2 * threads * maxSourceMoleculesInMemory` I think.